AzureFirewall/AzureFirewallApplicationRule.kql

```diff
@@ -0,0 +1,38 @@
+let applicationRuleAllowsHttp = AzureDiagnostics
+| where ResourceType == "AZUREFIREWALLS"
+| where Category == "AzureFirewallApplicationRule"
+| parse msg_s with * " request from " ip_source ":" port_source " to " ip_destination ":" port_destination ". Url: " url ". Action: " action ". Policy: " policy ". Rule Collection Group: " rulecollectiongroup "Rule Collection: " rulecollection ". Rule: " rule

```

Comment:
It would be highly beneficial if the field names correspond to [ASIM](https://docs.microsoft.com/en-us/azure/sentinel/normalization). The Azure Firewall would map to the Network and DNS schemas.

Reply:
I wasn't aware of the already existing ASIM parsers. I think Microsoft already did a way better job parsing the DNS entries for Azure Firewall in that parser. So I've removed my entry and pointed towards the ASIM one.